src/seedu/addressbook/data/person/Address.java

```diff
@@ -1,19 +1,26 @@
 package seedu.addressbook.data.person;
 
+import javax.swing.plaf.SliderUI;
+
 import seedu.addressbook.data.exception.IllegalValueException;
 
 /**
  * Represents a Person's address in the address book.
  * Guarantees: immutable; is valid as declared in {@link #isValidAddress(String)}
  */
 public class Address {
-
-    public static final String EXAMPLE = "123, some street";
-    public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses can be in any format";
-    public static final String ADDRESS_VALIDATION_REGEX = ".+";
+	public static final String EXAMPLE = "a/123, Clementi Ave 3, #12-34, 231534";
+    public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses must be in the following format: a/BLOCK, STREET, UNIT, POSTAL_CODE";
+    public static final String ADDRESS_VALIDATION_REGEX = "[\\d]+([A-Z])?, .+, #[\\d]{1,2}-[\\d]{2,3}, [\\d]{6}";
```

Comment:
Consider checking the validation inside each respective classes, for better SRP.